### DISCUSSION TITLE
Benefits page is not accessible

### DISCUSSION THREAD

**Discussion by DawidSzarek** [community/community#41865] [GitHub Education] [answered]
Hello

I have a problem accessing benefits from my profile. Each time I go to https://education.github.com/benefits I'm redirected to https://education.github.com/globalcampus/student without any message.

I was just verified yesterday and in reverification page it states that my request was indeed approved. I think I should have access to benefits, don't I?

Can someone help me with that?

**Comment by SilasRodrigues19** [comment]
@DawidSzarek Hi.

This redirection really happens. The link to see the benefits is here: [https://education.github.com/pack/offers](https://education.github.com/pack/offers)

**Comment by DSDmark** [comment, marked as answer]
Hi @DawidSzarek,

This is happening because you are already login in **GitHub student developer pack**. It means you are already verified. So, it's nothing you should worry about. You can access your list of benefits by click on ***explore offer button***

![image](https://user-images.githubusercontent.com/85744816/208115761-09fc23ca-9c6a-45bd-ac17-c537b6d72e9e.png)
